python/src/dynamixel_sdk/protocol2_packet_handler.py

```diff
@@ -434,47 +439,43 @@ def broadcastPing(self, port):
                     result = COMM_RX_CORRUPT
 
                     # remove header (0xFF 0xFF 0xFD)
-                    del rxpacket[0 : 3]
+                    del rxpacket[0: 3]
                     rx_length = rx_length - 3
 
             else:
                 # remove unnecessary packets
-                del rxpacket[0 : idx]
+                del rxpacket[0: idx]
                 rx_length = rx_length - idx
 
-        del rxpacket[:]; del rxpacket
+        # FIXME: unreachable code
```

Comment:
This should be addressed

Reply:
@roncapat 

According to the change above, 
this line is not needed, right?

Reply:
Yeah but I was mainly referring to the following return statement